### DISCUSSION TITLE
Trying to create a var for domains

### DISCUSSION THREAD

**Discussion by dkstar11q** [yourfavDev/go-brute#8] [Ideas]
Hello,

I am trying to create a variable in this project so it could take the first part of the domain (i.e. "qwerty.com" the username would be qwerty) with var := %domainname% 

I would like to use it in the username password list like this:

%domainname% %domainname%123

If you can help me how I can accomplish that it would be very helpful!

Thank you.

**Comment by yourfavDev** [comment]
If i understand correctly you would like to have specific combinations ( combos ) for each target?

**Reply by dkstar11q** [reply]
Yes, to be more specific, the input file containing the hostnames to brute:

asdf.net
qwerty.com
zxcvb.org

the user : password input file would look like:

%domainname% %domainname%123

which would result to try to login with the following credentials:

(asdf.net) asdf : asdf123
(qwerty.com) qwerty : qwerty123
(zxcvb.org) zxcvb : zxcvb123

**Reply by yourfavDev** [reply]
I dont think you understand what this program does.It is a ssh brute forcer not a website credential brute force.

That being said if u wish to create the feature you requested for ssh targets comment below.

**Reply by yourfavDev** [reply]
#9 We will continue the discussion here, thank you !

**Reply by dkstar11q** [reply]
Awesome, thank you for your time!! :)
